package.json

```diff
@@ -18,14 +18,17 @@
   },
   "devDependencies": {
     "@11ty/eleventy": "0.11.0",
+    "babel-eslint": "10.1.0",
```

Comment:
We need to not include this. We do not need this. The issue with the arrow functions can be fixed by changing the eslint config alone. Is the example I shared not working?

Reply:
Now that I'm looking at this it's kind of crazy how I installed a whole dependency just to get an arrow function to work.